examples/bert/README.md

```diff
@@ -14,6 +14,12 @@ The workflow in [bert_ptq_cpu_aml.json](bert_ptq_cpu_aml.json) is similar to the
 In order to use this example, `<place_holder>` in the `azureml_client` section must be replaced with the appropriate values for your
 AzureML workspace.
 
+#### Huggingface Model Optimization

```

Comment:
Just wondering, why don't we just update the existing one to fully use HF configs?

Reply:
bert_ptq_cpu.json is used for our e2e tests. I was thinking that `bert_ptq_cpu` will be updated frequently so just create a new one for huggingface case.

But the only difference is that the new one use huggingface evaluation but the old one not.

Reply:
Emm, it may be better to modify the existing one as there is too much json file which looks the new config bring redundancy.

Reply:
Updated.